test/creddit/test/client.clj

```diff
@@ -119,6 +119,16 @@
     (is (thrown? Exception (client/subreddit-top creddit-client "aww" "10" :month)))
     (is (thrown? Exception (client/subreddit-top creddit-client "aww" 10 "month")))))
 
+(deftest test-subreddit-hot
+  (testing "Retrieve hottest subreddit posts"
+    (is (= (with-fake-routes
+             {"https://www.reddit.com/r/aww/top/.json?limit=10"
```

Comment:
This should probably be `/hot` as well

Reply:
Indeed, I'm having some trouble getting everything to work after changing everything to /hot

Reply:
How are you getting on with this? What exactly is giving you trouble?

Reply:
The functionality works when changing every occurence of top to hot, but I'm having some trouble getting the test to pass